volttron/platform/agent/utils.py

```diff
@@ -234,8 +234,22 @@ def get_messagebus():
     return message_bus
 
 
+def is_web_enabled():
+    """Returns True if web enabled, False otherwise"""
+    is_web = os.environ.get('BIND_WEB_ADDRESS')
+    _log.debug("value from env {}".format(is_web))
+    if not is_web:
+        config = load_platform_config()
+        string_value = config.get('bind-web-address')
+        _log.debug("value from config {}".format(string_value))
+        if not is_web:
+            return False
+        return True
```

Comment:
This appears to return True if `is_web` is False, which is not the intent of the function. Am I reading this right?

Reply:
Fixed!